src/Symfony/Component/HttpFoundation/HeaderBag.php

```diff
@@ -20,14 +20,21 @@
  */
 class HeaderBag implements \IteratorAggregate, \Countable, \Stringable
 {
+    public const DEFAULT_CACHE_CONTROL_TARGET = '';
     protected const UPPER = '_ABCDEFGHIJKLMNOPQRSTUVWXYZ';
     protected const LOWER = '-abcdefghijklmnopqrstuvwxyz';
 
     /**
      * @var array<string, list<string|null>>
      */
     protected array $headers = [];
-    protected array $cacheControl = [];
```

Comment:
we need to add __get / __set to build and update this on the fly if somebody extended HeaderBag and manipulates those fields directly.